### PR TITLE
build: remove unused vscode settings

### PR DESCRIPTION
**Pull request by setchy**

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -1,13 +1,9 @@
 {
   "editor.codeActionsOnSave": {
-    "source.fixAll.eslint": "explicit",
-    "source.organizeImports.biome": "explicit",
-    "quickfix.biome": "explicit"
+    "quickfix.biome": "explicit",
+    "source.organizeImports.biome": "explicit"
   },
   "editor.defaultFormatter": "biomejs.biome",
```
Comment (afonsojramos) [inline .vscode/settings.json:6]:
```suggestion
  "editor.defaultFormatter": "biomejs.biome",
  "[typescript]": {
    "editor.defaultFormatter": "biomejs.biome"
  },
```
Unfortunately, you still need this even if you set it as the global default formatter.

File: .vscode/settings.json [hunk 2]
```diff
@@ -1,13 +1,9 @@
 {
   "editor.codeActionsOnSave": {
-    "source.fixAll.eslint": "explicit",
-    "source.organizeImports.biome": "explicit",
-    "quickfix.biome": "explicit"
+    "quickfix.biome": "explicit",
+    "source.organizeImports.biome": "explicit"
```
Comment (afonsojramos) [inline .vscode/settings.json:4]: I'm not sure about these, but I'll trust

**Review by afonsojramos — CHANGES_REQUESTED** [review]

File: .vscode/settings.json (hunk 1 above)
Comment (setchy) [reply .vscode/settings.json:6]: orly - been working fine without it for me 🤷‍♂️

Comment (afonsojramos) [reply .vscode/settings.json:6]: That depends on local settings, if you have a local default or not

**Pull request merged**